bin/pack.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
```

Comment:
Because npm does provide its own pack script, is there a possibility for this `bin/pack` script to override it?

Reply:
I believe all scripts defined in the bin folder are protected by the required `run` command to invoke them. 

`npm pack` vs `npm run pack`

Both commands are still working and don't appear to have any conflicts.